media/static-bundles.json

```diff
@@ -910,7 +910,8 @@
   "js": [
     {
       "files": [
-        "js/mozorg/contribute/contribute-nav.js"
+        "js/mozorg/contribute/contribute-nav.js",
+        "js/mozorg/contribute/contribute-survey.js"
```

Comment:
Instead of adding this to the page's base bundle, I'd suggest we create a new bundle just for the survey JS. We can then include that in the template, so we can control showing it via a switch. This lets us turn the survey on / off independently of a deployment.